pom.xml

```diff
@@ -145,7 +145,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-jar-plugin</artifactId>
-                    <version>3.0.2</version>
+                    <version>3.2.0</version>
```

Comment:
There was a separate issue for this upgrade, could you please look into this separately, if you'll be working on that as well? Thanks! :)

Please, revert this from this pull request.

Reply:
fixed

Reply:
Thanks! :)